cmd/workspace/pipelines/pipelines.go

```diff
@@ -84,37 +84,6 @@ func newCreate() *cobra.Command {
 
 	cmd.Flags().Var(&createJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().BoolVar(&createReq.AllowDuplicateNames, "allow-duplicate-names", createReq.AllowDuplicateNames, `If false, deployment will fail if name conflicts with that of another pipeline.`)
```

Comment:
This seems to be quite a big of backward incompatible change, shall we call it out at least in release notes?

Reply:
They were added by mistake in #3769.

It was corrected in a later SDK release and that's now picked up.

Reply:
These flags were never intended to be added. Both the jobs create and pipeline create commands should always take a `--json` because the payloads are too complex to be specified via flags.